data_extractor/google_search_history/__init__.py

```diff
@@ -0,0 +1,55 @@
+""" Script to extract info from Google Browser History """
+
+__version__ = '0.1.0'
+
+import json
+import datetime as DT
+import numpy as np
+import pandas as pd
+import re
+
+
+def process(data):
+    """Return relevant data from browser history pre and post specific date
+    Args:
+        data: BrowserHistory.json file
```

Comment:
When I download my Google Chrome data, it comes in a zip file. I think the data donor will upload the zipfile (and not unpack it to select the json-file). So probably the code should accept a zipfile object as input.